idea/testData/inspectionsLocal/replaceJavaIntegerToStringWithMemberInspection/replaceIntToString.kt

```diff
@@ -0,0 +1,5 @@
+// WITH_RUNTIME
```

Comment:
Some test when inspection isn't applicable might be useful, I suggest to do one with invalid code `Integer.toString(<caret>)`, as `isToString()` implicitly suggests that this case won't be resolved to a needed qualified name.

Reply:
I have added this test, adding the directive:
```
// ERROR: None of the following functions can be called with the arguments supplied: <br>public open fun toString(i: Int): String! defined in java.lang.Integer<br>public open fun toString(i: Int, radix: Int): String! defined in java.lang.Integer
```
Is this correct? It looks fishy to me.

Reply:
I agree it doesn't look very nice to have a full error message in the test data. Maybe only message code should be there (no need to implement this in this PR).

It was decided to list errors in the test data file to prevent a situation when an intention is not applicable not because of program logic, but because of compile errors in the test data file.